test/test_config/good_conjurrc

```diff
@@ -1,5 +1,5 @@
 ---
 account: accountname
-plugins: ['foo', 'bar']
 appliance_url: https://someurl/somepath
 cert_file: "/cert/file/location"
+plugins: ['foo', 'bar']
```

Comment:
What are these plugins?

Reply:
the purpose of plugins in conjur I am not sure. Previously they were for host factories, layers etc now that they are part of the core cli functionality, don't know why we still need them but if a user can input a value then we should support that usecase